dev/Scroller/APITests/ScrollerLayoutTests.cs

```diff
@@ -80,8 +80,8 @@ public void BasicMargin()
                 SnapPointsMode.Ignore,
                 hookViewChanged: true,
                 isAnimationsEnabledOverride: null,
-                c_defaultUIScrollerContentWidth + 2 * c_Margin - c_defaultUIScrollerWidth,
-                c_defaultUIScrollerContentHeight + 2 * c_Margin - c_defaultUIScrollerHeight);
+                expectedFinalHorizontalOffset: c_defaultUIScrollerContentWidth + 2 * c_Margin - c_defaultUIScrollerWidth,
```

Comment:
Are these changes related to the main PR?

Reply:
These errors started showing up in the build for some reason and i had to fix them.